# Generated by Django 4.2.5 on 2023-09-12 06:50

from django.db import migrations, models
import django.db.models.deletion
import hacker_exception_app.models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Order',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('delivery_name', models.CharField(max_length=200)),
                ('delivery_address', models.TextField()),
                ('amount', models.FloatField()),
                ('status', models.CharField(blank=True, choices=[('NEW', 'New'), ('DISPATCHED', 'Dispatched'), ('DELIVERED', 'Delivered')], default=hacker_exception_app.models.OrderStatus['NEW'], max_length=50, null=True)),
                ('created_date', models.DateTimeField(auto_now_add=True)),
            ],
        ),
        migrations.CreateModel(
            name='Payment',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('amount', models.FloatField()),
                ('created_date', models.DateTimeField(auto_now_add=True)),
                ('order', models.OneToOneField(null=True, on_delete=django.db.models.deletion.SET_NULL, to='hacker_exception_app.order')),
            ],
        ),
    ]
